pipelines/HSC/DRP-Prod.yaml

```diff
@@ -101,6 +101,7 @@ subsets:
       - makePsfMatchedWarp
       - selectDeepCoaddVisits
       - assembleCoadd
+      - assembleCellCoadd
```

Comment:
Is there a way to include these only if `parameters.use_cell_coadds` is `true`?

Reply:
That all feels a little too magical to me, especially since I think sometimes it might be desirable to build the cell-based coadds without using them (that's what we were going to do first, right?).

If you're worried about the pain of dropping `assembleCellCoadd`  from subsets in many downstream pipelines, using `labeledSubsetModifyMode: EDIT` and `exclude: assembleCellCoadd` in the pipeline import section can probably help a lot with that (see pipelines that import `base-v2.yaml` for examples).